Bindings/bindings.json

```diff
@@ -6,473 +6,504 @@
+        }
+      ]
+    },
+    {
+      "type": "notificationHub",
+      "displayName": "Azure Notification Hub",
+      "direction": "out",
+      "settings": [
+        {
+          "name": "tagExpression",
+          "value": "string",
+          "defaultValue": "tag1||tag2",
+          "required": false,
+          "label": "Optional.Tag expression for sending push notification. Leave this blank to send broadcast",
+          "help": "Tag expressions enable you to target specific sets of devices/registrations, when sending a push notification through Notification Hubs."
         },
         {
-            "type": "easyTable",
-            "displayName": "Azure Mobile Easy Table Record",
-            "direction": "out",
-            "defaultParameterName": "record",
-            "parameterNamePrompt":"Name the record for use in your code",
-            "settings": [
-                {
-                    "name": "tableName",
-                    "value": "string",
-                    "defaultValue": "MyTable",
-                    "required": false,
-                    "label": "To which table should the data be sent?",
-                    "help": "Please enter the name of the table within your Mobile App. If you are using a strongly-typed parameter in C# this property is not required."
-                }
-            ]
+          "name": "hubName",
+          "value": "string",
+          "defaultValue": "MyNotificationHubName",
+          "required": true,
+          "label": "Name of the Notification hub to be used for sending push notifications",
+          "help": "Please enter the name of the Notification Hub"
         },
         {
-            "type": "easyTable",
-            "displayName": "Azure Mobile Easy Table Record",
-            "direction": "in",
-            "defaultParameterName": "record",
-            "parameterNamePrompt":"Name the record for use in your code",
-            "settings": [
-                {
-                    "name": "tableName",
-                    "value": "string",
-                    "defaultValue": "MyTable",
-                    "required": false,
-                    "label": "To which table should the data be sent?",
-                    "help": "Please enter the name of the table within your Mobile App. If you are using a strongly-typed parameter in C# this property is not required."
-                },
-                {
-                    "name": "id",
-                    "value": "string",
-                    "defaultValue": "{id}",
-                    "required": true,
-                    "label": "What is the Id for the record?",
-                    "help": "Please enter the id for the record to retrieve."
-                }
-            ]
+          "name": "connection",
+          "value": "string",
+          "resource": "NotificationHub",
+          "required": true,
+          "label": "Which NotificationHub namespace should be used?",
```

Comment:
setting is connection. The value of the connection string is for the NH namespace